static/docs/commands-reference/metrics/add.md

```diff
@@ -13,14 +13,20 @@ positional arguments:
 
 ## Description
 
-Sets a special flag in the relevant [DVC-file](/doc/user-guide/dvc-file-format)
-to identify a specified output as a metric file. Alternatively, an output file
-can be marked as a metric via `-M` or `-m` parameter of the `dvc run` command.
+Sets a special field, in the [DVC-file](/doc/user-guide/dvc-file-format) that
```

Comment:
probably should remove `special` here as well

Reply:
Removed and reviewed throughly in my last commit.